### ISSUE TITLE
Why the struct PKCS7.remove 's second parameter use option Int, not Int

### ISSUE THREAD

**Issue by neil-wu** [krzyzanowskim/CryptoSwift#80]
Actually, I search the entire project and only find one place to call this function with nil blocksize parameter. 
(AES.swift,func decrypt, line 212)
One more thing, I use pyCrypto to gen the encrypt data, and load it in swift. My python pad function like that:
![image](https://cloud.githubusercontent.com/assets/5682948/8158360/ff56a42c-138e-11e5-92c5-633e8e5603a9.png)

I just add some zero value. While this may lead to crash in swift. Because PKCS7.add function is different from my python version.
How about modify the PKCS7.remove function like the following code?
```javascript
public func remove(bytes: [UInt8], blockSize:Int) -> [UInt8] {
    //var padding = Int(bytes.last!) // last byte
    var bs = blockSize ?? AES.blockSize //modify
    var padding = Int(bs - (bytes.count)%bs ) //modify
    if padding >= 1 { //TODO: need test for that, what about empty padding
        return Array(bytes[0..<(bytes.count - padding)])
    }
    return bytes
}
```

**Comment by krzyzanowskim** [comment]
why?

**Comment by neil-wu** [comment]
may the pad function be different from each other in different programming language?

**Comment by krzyzanowskim** [comment]
it's standard. if you think it's not working properly, please refer to the standard and provide some failing tests

**Comment by neil-wu** [comment]
OK. Thanks a lot.  :）

**Comment by krzyzanowskim** [comment]
Resolved by #57
